### ISSUE TITLE
JSON to HTML User Interface converter.  

### ISSUE THREAD

**Issue by prabinzz** [nepalihackers/ideas#7]
Create a program that is capable of converting organized  JSON  data into HTML ui.

suppose i have folloing thing in json file.
```
{
"title": "Title name.",
"body": "Some texst."
}
```
Program should be abel to convert this into appropriate HTML ui.

**Comment by amitness** [comment]
Cool idea. You could use a base bootstrap layout and then link bootstrap components to the JSON properties.

**Comment by yalu** [comment]
ClojureScript has

https://github.com/r0man/sablono - express React compnents as EDN(which is like json - pure data)

https://github.com/weavejester/hiccup - same thing for pure HTML

Since it is pure data like Json you move it across the wire with Transit
http://blog.cognitect.com/blog/2014/7/22/transit

**Comment by prabinzz** [comment]
@amitness Bootstrap layout  will be great. There in no point in making ordinary html layout, we can do that manually.

**Comment by prabinzz** [comment]
@yalu  Yeah  that will work too.  But i think,  we should use JSON  it has more support and we all are familier with it .

**Comment by prabinzz** [comment]
@yalu any language will be fine,  as long as it create desired output.. 👍
